tools/docker-images/clp-package/build.sh

```diff
@@ -22,3 +43,13 @@ then
 fi
 
 "${build_cmd[@]}"
+
+if [[ -s "$temp_iid_file" ]]; then
+    new_image_id="$(<"$temp_iid_file")"
+    echo "$new_image_id" > "$iid_file"
+
+    user="${USER:-$(id -un 2>/dev/null || whoami 2>/dev/null || echo unknown)}"
```

Comment:
There's one edge case we missed to handle -

When the username cannot be found on the system (e.g., the username is not specified in `/etc/passwd`), the user id would still be printed by `id -un` command. However, since the `id -un` command exits with non-zero code, the other command would continue to run, which may cause line breaks to show up in the middle of the `user` string. e.g.,

```
id -un: Prints `123\n`; Exits with non-0
whoami: Prints nothing; Exits with non-9
echo unknown: Prints "unknown"

then `user` becomes: `123\nunknown`
```

@coderabbitai create an issue to track this